### PR TITLE
Add a workaround for torch<1.11.0 bug with tensor indexing

### PR DESCRIPTION
**Pull request by vshampor**
### Changes

For torch < 1.11.0, additionally wrap tensor indices for torch.Tensor in a tuple so that indexing a tensor using `TracedTensor` indices has correct behaviour.

### Reason for changes

Fixes incorrect behaviour of NNCF for clients with torch < 1.11.0.

### Related tickets

82065

### Tests

TBA

### CODE REVIEW

**Comment by vinnamkim** [comment]
[upstream patch](https://github.com/pytorch/pytorch/pull/67202 ) was applied `torch==1.11.0`.

**Comment by vinnamkim** [comment]
For `torch<1.11.0`, this PR patches `__getitem__` operator as same as https://github.com/pytorch/pytorch/issues/52721#issuecomment-804773009.

File: nncf/torch/__init__.py [hunk 1]
```diff
@@ -17,15 +17,15 @@
...
+CURRENT_TORCH_VERSION = parse_version(torch.__version__)
+if parse_version(BKC_TORCH_VERSION).base_version != CURRENT_TORCH_VERSION.base_version:
     import warnings
     warnings.warn("NNCF provides best results with torch=={bkc}, "
                    "while current torch version is {curr} - consider switching to torch=={bkc}".format(
          bkc=BKC_TORCH_VERSION,
          curr=torch.__version__
     ))
-elif torch_version < '1.5.0' or torch_version == '1.8.0':
+elif CURRENT_TORCH_VERSION < parse_version('1.5.0') or CURRENT_TORCH_VERSION.base_version == '1.8.0':
```
Comment (wonjuleee) [inline nncf/torch/__init__.py:28]: should first CURRENT_TORCH_VERSION be CURRENT_TORCH_VERSION.base_version?

**Comment by ljaljushkin** [comment]
@vshampor, is it actual since we bumped torch version to 1.12?

**Comment by vshampor** [comment]
> @vshampor, is it actual since we bumped torch version to 1.12?

Not really.

**Pull request closed without merging**